pallets/block-reward/src/tests.rs

```diff
@@ -18,7 +18,7 @@
 
 use super::{pallet::Error, Event, *};
 use frame_support::{assert_noop, assert_ok, traits::OnTimestampSet};
-use mock::*;
+use mock::{Balance, *};
```

Comment:
Any reason to explicitly import `Balance`?

Reply:
had to use to because `Balance` was ambiguous.

Reply:
have to make it unambiguous, defined in `lib.rs` and `mock.rs` both.